docs/site/Transactions.md

```diff
@@ -23,10 +23,10 @@ is attached to one of the following connectors:
 - [PostgreSQL connector](PostgreSQL-connector.html)
 - [SQL Server connector](SQL-Server-connector.html)
 - [Oracle connector](Oracle-connector.html)
-- [DB2 Connector](DB2-connector.html)
 - [DashDB Connector](DashDB.html)
-- [DB2 iSeries Connector](DB2-iSeries-connector.html)
-- [DB2 for z/OS connector](DB2-for-z-OS.html)
+- [IBM Db2 (for Linux, Unix, Windows) Connector](DB2-connector.html)
+- [IBM Db2 for i connector](DB2-for-i.html)
```

Comment:
I think the link is still `(DB2-iSeries-connector.md)` as we haven't changed the permalink of the page.

Reply:
please ignore my comment above, as https://github.com/strongloop/loopback-next/pull/4409#discussion_r393769211 explained it.

Reply:
hm. I don't think the *.md files exist either, because those are coming from the readme fetching from the [loopback.io script](https://github.com/strongloop/loopback.io/blob/gh-pages/update-readmes.sh). 

Same as previous comment, we're removing "db2 iseries connector" and replace it with "db2 for i""